components.json

```diff
@@ -57,6 +57,10 @@
 			},
 			"option": "default"
 		},
+		"cil": {
```

Comment:
You have to place this further down below (under C++, I think).
These here are special languages for Prism.